### ISSUE TITLE
ERRO[0000] Cannot get exit code: failed to get journal cursor: failed to get cursor: cannot assign requested address 

### ISSUE THREAD

**Issue by eblot** [containers/podman#11572]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

When executing basic `podman run` command, I keep getting this error message:
```
ERRO[0000] Cannot get exit code: failed to get journal cursor: failed to get cursor: cannot assign requested address 
```
I've been unable to find a trouble shooting guide for this issue.

**Steps to reproduce the issue:**

1. `podman run --rm -ti alpine /bin/true; echo $?`

**Describe the results you received:**

```
ERRO[0000] Cannot get exit code: failed to get journal cursor: failed to get cursor: cannot assign requested address 
127
```

**Describe the results you expected:**

No error, and return code = 0

**Additional information you deem important (e.g. issue happens only occasionally):**

It used to happen every other run, now that I've upgraded several SW packages to their latest version, it always occurs.

**Output of `podman version`:**

```
Version:      3.3.1
API Version:  3.3.1
Go Version:   go1.16.7
Built:        Thu Jan  1 01:00:00 1970
OS/Arch:      linux/amd64
```

**Output of `podman info --debug`:**

```
host:
  arch: amd64
  buildahVersion: 1.22.3
  cgroupControllers: []
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: 'conmon: /usr/bin/conmon'
    path: /usr/bin/conmon
    version: 'conmon version 2.0.25, commit: unknown'
  cpus: 32
  distribution:
    distribution: debian
    version: "11"
  eventLogger: journald
  hostname: keembay
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 502
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  kernel: 5.14.0-trunk-amd64
  linkmode: dynamic
  memFree: 30290731008
  memTotal: 33586475008
  ociRuntime:
    name: crun
    package: 'crun: /usr/bin/crun'
    path: /usr/bin/crun
    version: |-
      crun version 0.17
      commit: 0e9229ae34caaebcb86f1fde18de3acaf18c6d9a
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
  os: linux
  remoteSocket:
    exists: true
    path: /run/user/502/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: false
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: 'slirp4netns: /usr/bin/slirp4netns'
    version: |-
      slirp4netns version 1.0.1
      commit: 6a7b16babc95b6a3056b33fb45b74a6f62262dd4
      libslirp: 4.4.0
  swapFree: 20106440704
  swapTotal: 20106440704
  uptime: 30m 48.01s
registries:
  search:
  - docker.io
store:
  configFile: /home/eblot/.config/containers/storage.conf
  containerStore:
    number: 10
    paused: 0
    running: 0
    stopped: 10
  graphDriverName: overlay
  graphOptions:
    overlay.mount_program:
      Executable: /usr/bin/fuse-overlayfs
      Package: 'fuse-overlayfs: /usr/bin/fuse-overlayfs'
      Version: |-
        fusermount3 version: 3.10.3
        fuse-overlayfs: version 1.4
        FUSE library version 3.10.3
        using FUSE kernel interface version 7.31
  graphRoot: /home/eblot/.local/share/containers/storage
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 16
  runRoot: /run/user/502/containers
  volumePath: /home/eblot/.local/share/containers/storage/volumes
version:
  APIVersion: 3.3.1
  Built: 0
  BuiltTime: Thu Jan  1 01:00:00 1970
  GitCommit: ""
  GoVersion: go1.16.7
  OsArch: linux/amd64
  Version: 3.3.1
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
Listing... Done
podman/testing,unstable,now 3.3.1+ds2-1 amd64 [installed]
podman/stable 3.0.1+dfsg1-3+b2 amd64
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide? (https://github.com/containers/podman/blob/master/troubleshooting.md)**

Yes

It seems to be the latest version, and the trouble shooting guide does not seem to give any advice on this specific error.

**Additional environment details (AWS, VirtualBox, physical, etc.):**

* Physical: AMD Ryzen3 5950x, 32GB ram, single SSD, no network mount
* Debian 11, with default kernel (5.10) and with experimental kernel (5.14), using crun
* Docker installed (and running ok) docker.io/stable,testing,unstable,now 20.10.5+dfsg1-1+b5 amd64 [installed] 

Please note that when replacing `crun` with `runc` I get a totally different error

```
Error: container_linux.go:367: starting container process caused: error adding seccomp filter rule for syscall bdflush: permission denied: OCI permission denied
```
for which the recommended workaround seems to be to use ... `crun` :-)

**Comment by Luap99** [comment]
You need to change your `eventLogger` to `file` in the containers.conf config file.

**Comment by eblot** [comment]
thanks a lot @Luap99, it seems to make the job.

**Comment by Luap99** [comment]
Starting with version 3.4 we should be able to fall back to `file` automatically when we cannot use journald.

**Comment by caizixian** [comment]
@Luap99 I'm still seeing the issue on Podman 3.4.4 shipped with Ubuntu 22.04
``` console
$ podman version
Version:      3.4.4
API Version:  3.4.4
Go Version:   go1.17.3
Built:        Thu Jan  1 00:00:00 1970
OS/Arch:      linux/amd64
```

I'm able to workaround the issue by setting `events_logger = "file"` in containers.conf.

**Comment by Luap99** [comment]
This version is no longer supported by us upstream, please try with at least v4.2 and see if this fixes your problem. If not file a new bug with a reproducer.
